### PR TITLE
finished

### PR DESCRIPTION
**Pull request by nickMon85**

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -14,25 +14,105 @@
...
+  }
+  
+
+}
+//1.Create a function 
+//2. Separate string and boolean values
+//3. Divide numbers by 3
+//4. Return Only Remainders
+
+//Come back to can't figure out
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:116]: This one would be a good one to come back too. If you have an array and you need only some values back HIGHLY recommend the .filter method.

File: code-challenges.test.js [hunk 2]
```diff
@@ -41,11 +121,38 @@ const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
...
+  })
+})
+
+const cubeSum =(arr)=>{
+  //cube every number!
+ let newCube = arr.map(function(e){
+   return e*e*e
+ })
+ //Added the sum of the cubed numbers
+ let sum = newCube.reduce((partialSum,a)=> partialSum + a,0);
```
Comment (WalkerTexas-coder) [inline code-challenges.test.js:152]: Good use of .reduce

File: interview-questions.md [hunk 3]
```diff
@@ -7,44 +7,79 @@ Then, research the question to expand on your answer. Even if you feel you have
...
+1.)As a User, I would like to see my private to do list. 
+2.)As a User, I would like to see a calender due dates for my to do list.
+3.)As a User, I would like to have a finished or done button, for once an item is accomplished
+4.)As a User, I would like to have an option of a reminder.
+5.) As a User, I would like to hold the list of finished items or delate.
+6.)As a User, I would like to organize the list in level of importance. Maybe color code!
+7.)As a User, I would like to organize a list as either routine or event.
+8.)As a User, I would like to edit or update 
+9.)As a User, I would like to add a list on an item for any necessary tools or equipment.
+10.)As a User, I would like a small happy sound or song if everything has been accomplished.
```
Comment (WalkerTexas-coder) [inline interview-questions.md:85]: This is close! What you created feels very similar to a wish list. If you are looking to get into the Dev industry I recommend you brush up on the idea of user stories and get used to what they look like before interviews.

File: rails-commenting.rb [hunk 4]
```diff
@@ -54,15 +54,15 @@ def destroy
...
       redirect_to blog_posts_path
     else
-      # ---8)
+      # ---8)This second line redirect the user to @post method using a redirect_to call. 
       redirect_to blog_post_path(@post)
     end
   end
 
-  # ---9)
+  # ---9) Private method this is not accessible to outside of the class. A private method is defined by using a private keyword.
```
Comment (WalkerTexas-coder) [inline rails-commenting.rb:62]: Good stuff!

**Review by WalkerTexas-coder — APPROVED** [review]
Good effort man! You've really grown as a developer! Congratulations on wrapping this up.

**Review by SunkissedQueen — APPROVED** [review]
Great work! So the user stories are meant to show exactly what the user can see on the screen with a finished product. So the phrasing should be "As a user I can see my private to do list."

**Pull request merged**